lib/ae_mdw/names.ex

```diff
@@ -504,6 +559,39 @@ defmodule AeMdw.Names do
     |> Enum.map(&render_name_info(state, &1, opts))
   end
 
+  defp render_claim(state, plain_name, {{height, _mbi} = block_index, txi}) do
+    tx_hash = Txs.txi_to_hash(state, txi)
+    block_hash = Blocks.block_index_to_hash(state, block_index)
+
+    claim_aetx =
+      case Db.get_tx_data(tx_hash) do
+        {_block_hash, :name_claim_tx, signed_tx, _tx_rec} ->
+          :aetx_sign.tx(signed_tx)
+
+        {_block_hash, :contract_call_tx, _signed_tx, _tx_rec} ->
+          state
+          |> Contracts.fetch_int_contract_calls(txi, "AENS.claim")
+          |> Enum.find_value(fn Model.int_contract_call(tx: aetx) ->
+            {:name_claim_tx, tx} = :aetx.specialize_type(aetx)
+            name = :aens_claim_tx.name(tx)
+
+            plain_name == name
```

Comment:
It would return `true` instead of the `tx`. May we have also a coverage for internal claim?

Reply:
Good find, thanks. Done